include/megaapi.h

```diff
@@ -18285,6 +18318,25 @@ class MegaApi
         */
         void sendBackupHeartbeat(MegaHandle backupId, int status, int progress, int ups, int downs, long long ts, MegaHandle lastNode);
 
+        /**
+         * @brief Allow fetch google ads
```

Comment:
```suggestion
         * @brief Fetch Google ads
```
Additionally, the doc about the request and its params when API_OK is missing.